src/main/java/useCases/Timer/TimerInteractor.java

```diff
@@ -46,13 +42,13 @@ void startTimer(){
     }
 
     /**
-     * Stops the timer countdown and holds the duration of the timer constant till the user doesn’t press start again
+     * Stops the timer countdown and holds the duration of the timer constant till the user does not press start again
      *     Updates the remainingDuration variable in Timer Class
      */
     void pauseTimer(){}
 
     /**
-     * When user ends the timer session, output “end of session”
+     * When user ends the timer session, output end of session
      *     Is also automatically called when the Time on the clock runs out
      */
     void endTimer(){}
```

Comment:
Shouldn't the output message be implemented here so it can be passed to the Presenter to output?